api/tx_args.go

```diff
@@ -168,7 +168,7 @@ func (args *SendTxArgs) setDefaults(ctx context.Context, b Backend) error {
 			args.MaxPriorityFeePerGas = (*hexutil.Big)(fixedGasPrice)
 		}
 		if args.MaxFeePerGas == nil {
-			fixedBaseFee := new(big.Int).SetUint64(params.BaseFee)
+			fixedBaseFee := new(big.Int).SetUint64(params.ZeroBaseFee)
```

Comment:
@kjeom @jimni1222  We need a TODO tag here. As far as I know, we gonna set `2 * SuggestPrice` for `args.MaxFeePerGas`.

Reply:
@aidan-kwon Thanks. I added the tag and will be implemented in next PR